static/js/subs.js

```diff
@@ -42,6 +37,33 @@ exports.is_sub_already_present = function (sub) {
     return exports.row_for_stream_id(sub.stream_id).length > 0;
 };
 
+exports.update_left_panel_row = (sub) => {
+    const row = exports.row_for_stream_id(sub.stream_id);
+
+    if (row.length === 0) {
+        return;
+    }
+
+    blueslip.info(`Updating row in left panel of stream settings for: ${sub.name}`);
```

Comment:
This should probably be `blueslip.debug`; we don't need this cluttering the JS console for folks not debugging this area.

Reply:
Fixed.